test/extended/router/OWNERS

```diff
@@ -1,14 +1,25 @@
 approvers:
-  - danehans
   - frobware
   - knobunc
   - Miciah
```

Comment:
Please keep Miciah, miheer and frobware as they in NetEdge team.

Reply:
@Thealisyed i still don't see the above change. Can you please add these 3 names in the approvers and reviewers ?

Reply:
Not sure why this comment was marked as resolved.

Reply:
@miheer I can see that Miciah, miheer and frobware are in both approvers and reviewers in the PR?

Reply:
@miheer - I agree, they are in the original file and not removed.